### PR TITLE
Modify async_events.md grammar

### LINKED ISSUE #363: Minor grammatical changes to documentation
It's a bit hard to follow some of the documentation early in the morning.

### PR DESCRIPTION
**Pull request by ddproxy**
Had a slight problem reading documentation early in the morning, realized there were some grammatical changes that would make this a bit easier to read.

Resolves #363 within the `async_events.md` document.

### CODE REVIEW

**Comment by makasim** [comment]
Thank you.

**Comment by ddproxy** [comment]
@makasim - Pinged you in Gitter, btw.
